R/stan_module.R

```diff
@@ -10,17 +10,17 @@
 #' @slot inits List with the initial values of the stan model.
 #' @export
 StanModule <- setClass(
-    "StanModule",
-    representation(
-        functions = "character",
-        data = "character",
-        parameters = "character",
-        transformed_parameters = "character",
-        model = "character",
-        priors = "list",
-        generated_quantities = "character",
-        inits = "list"
-    )
+  "StanModule",
+  representation(
+    functions = "character",
+    data = "character",
+    parameters = "character",
+    transformed_parameters = "character",
+    model = "character",
+    priors = "list",
+    generated_quantities = "character",
+    inits = "list"
+  )
 )
```

Comment:
I see here that you have changed the spacing of the code. For better readability of the R code, it is principled that we use the same "scale" of spacing to all the R files. At the begging, Craig suggested that we used spacing of "4" instead of "2".  To change this to your laptop, go to `Addins` in your R studio --> set style --> change the "styler::tidyverse_style()" to styler::tidyverse_style( indent_by = 4 )" in the window that appears and ok--> `addins` --> style active file (or package).

Reply:
Thanks. The indentation has been changed